components/eamxx/src/python/libpyscream/pyfield.hpp

```diff
@@ -24,7 +26,9 @@ struct PyField {
     f.allocate_view();
   }
 
-  pybind11::array get () const {
+  // FIXME: how to return ndarray?
+  template <typename X, typename Y, typename Z>
+  nb::ndarray<X,Y,Z> get () const {
```

Comment:
I would rename this as `array_host` or something. Better to be explicit on the fact that it's host data, so the user remembers to sync later. Also, we may later add `array_dev`, to allow grabbing dev pointers directly.